### ISSUE TITLE
Move key tree

### ISSUE THREAD

**Issue by dsmalko** [glebm/i18n-tasks#116]
Is there a way to move key trees in multiple locales? Like change en.posts -> en.articles

**Comment by dsmalko** [comment]
What if I want to move en.company.posts -> en.admin.posts?

**Comment by glebm** [comment]
@dsmalko That would be a feature i18n-tasks doesn't support yet.

**Comment by glebm** [comment]
*My thoughts on what this should look like (from a comment in #126).*

### Move / rename / merge keys

`i18n-tasks mv <pattern> <key>` is a versatile task to merge-move nodes.
All nodes (leafs or subtrees) matching [`<pattern>`](#key-pattern-syntax) are merged together and
merge-moved to `<key>`.

Rename a node (leaf or subtree):

```console
$ i18n-tasks mv user account
```

Move a node:

```console
$ i18n-tasks mv user_alerts user.alerts
```

Move the children one level up:

```console
$ i18n-tasks mv 'alerts.{:}' '\1'
```

Merge-move multiple nodes:

```console
$ i18n-tasks mv '{user,profile}' account
```
Merge nodes into parent:

```console
$ i18n-tasks mv 'pages.{a,b}' pages
```

**Comment by glebm** [comment]
Released in [v0.9.8](https://github.com/glebm/i18n-tasks/releases/tag/v0.9.8)! :santa:
